scenarios/web/small/core/2.1/mvc/mvc.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RuntimeFrameworkVersion>2.1.0-preview1-26115-03</RuntimeFrameworkVersion>
```

Comment:
I believe `RuntimeFrameworkVersion` and `NETStandardImplicitPackageVersion` should be removed.  Does the app still work without them?

Reply:
I doesn't restore successfully without it. This was part of the 2.1 template

Reply:
I will investigate.  If this attribute is needed, I believe it's a symptom of a bigger problem (like a version mismatch between ASP.NET and CoreFx).

Reply:
`RuntimeFrameworkVersion` is required since the Preview2 CLI builds are currently incoherent (bundled runtime doesn't match NuGetFallbackFolder).  This will be fixed when we start using ProdCon builds for Preview2.

Everything seems to work fine if `NETStandardImplicitPackageVersion` is removed, so let's remove it from the PR.